app/moves/controllers/dashboard.js

```diff
@@ -0,0 +1,28 @@
+const { find, reject } = require('lodash')
+
+function setTotalMoves(req, res, next) {
+  const totalMoves = {
+    label: 'moves::dashboard.filter.total',
+    filter: 'total',
+    href: find(res.locals.moveTypeNavigation, { filter: 'proposed' }).href,
+  }
+  totalMoves.value = res.locals.moveTypeNavigation.reduce(
+    (accumulator, moveType) => {
+      return (accumulator += moveType.value)
+    },
+    0
+  )
+  res.locals.moveTypeNavigation = reject(res.locals.moveTypeNavigation, {
```

Comment:
I think it might be cleaner to use a separate variable name here rather than override `moveTypeNavigation`.

The value is used in a separate view and it feels like it will cause us less issues in the future. It's also no longer the "navigation" if you think about it. Maybe it's more a summary of proposed moves?